packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/scale.routing.js

```diff
@@ -0,0 +1,42 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.kubernetes.details.nodepools.scale',
+    {
+      url: '/scale?nodePoolId',
+      views: {
+        modal: {
+          component: 'pciProjectKubernetesNodePoolsScaleComponent',
+        },
+      },
+      layout: 'modal',
+      params: {
+        nodePoolId: null,
+        nodePoolName: null,
```

Comment:
Can't we fetch the name from the `nodePoolId` ?

Reply:
I am not sure to understand.
Name and Id are passed from parent state.

Reply:
So if we want to access the url directly with the id we have to pass the name as well ? 
Also this means that if we need something else like description, we would pass it as well through parameters ? Not sure it is the best way to achieve things that to pass every info through params, id should be sufficient to get all the required info

Reply:
Thanks for the remark @marie-j.
It's a mistake from my side, i missed to remove the `nodePoolName` param (no needed)